schedule/yast/autoyast/autoyast_tftp.yaml

```diff
@@ -30,7 +30,7 @@ test_data:
     - path: /etc/hosts
       entries:
         - '10.226.154.19\tnew.entry.de h999uz'
-    - path: /etc/chrony.conf
+    - path: /etc/chrony.d/pool.conf
```

Comment:
please verify as well for https://openqa.suse.de/tests/10492453

Reply:
Done.